routers/repo/issue.go

```diff
@@ -1237,6 +1237,67 @@ func UpdateIssueAssignee(ctx *context.Context) {
 	})
 }
 
+// updatePullReviewRequest change pull's request reviewers
+func updatePullReviewRequest(ctx *context.Context, isReRequest bool) {
+	issues := getActionIssues(ctx)
+	if ctx.Written() {
+		return
+	}
+
+	reviewID := ctx.QueryInt64("id")
+	action := ctx.Query("action")
+
+	if action != "" {
+		ctx.ServerError("updatePullReviewRequest", fmt.Errorf("action should not be \"%s\"", action))
+		return
+	}
+
+	for _, issue := range issues {
+		if issue.IsPull {
+
+			if isReRequest && issue.PosterID != ctx.User.ID {
+				ctx.ServerError("updatePullReviewRequest", fmt.Errorf("%d is not poster of %d PR", ctx.User.ID, issue.ID))
+			}
+
+			reviewer, err := models.GetUserByID(reviewID)
+			if err != nil {
+				ctx.ServerError("GetUserByID", err)
+				return
+			}
+
+			valid, err := models.CanBeAssigned(reviewer, issue.Repo, issue.IsPull)
+			if err != nil {
+				ctx.ServerError("canBeAssigned", err)
+				return
+			}
+			if !valid {
+				ctx.ServerError("canBeAssigned", models.ErrUserDoesNotHaveAccessToRepo{UserID: reviewID, RepoName: issue.Repo.Name})
+				return
+			}
+
+			err = issue_service.ToggleReviewRequest(issue, ctx.User, reviewer)
```

Comment:
Sorry, That's because the code stly is copy from Assign , but after thinking, I think your suggestion is necessary , because If more than two request add or remove to one reviewer in a same time , It will become to a bug, I will change them,  Thanks.